test/opal/test_strict.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: false
+require "test/unit"
+
+class TestStrict < Test::Unit::TestCase
+  def test_strict_mode
+    assert_equal(0, 0.times.size)
```

Comment:
If you revert the oneline fix in `lib/opal/nodes/iter.rb`, `minitest` should fail.